### PR TITLE
Add notifications to MatterMost

### PR DESCRIPTION
**Pull request by nicolasbock**
Closes: canonical/hotsos#189

### CODE REVIEW

**Comment by nicolasbock** [comment]
@dosaboy We need to add some secrets to this repo before this will work.

**Review by sombrafam — CHANGES_REQUESTED** [review]
There's a typo in the filename 'notofications'

**Review by dosaboy — CHANGES_REQUESTED** [review]
Filename should be notifications.yaml

File: .github/workflows/notofications.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Notifications
+on:
+  issues:
+  pull_request:
+    branches: [ main ]
```
Comment (dosaboy) [inline .github/workflows/notofications.yaml:5]: types: [ready_for_review]

**Review by dosaboy — CHANGES_REQUESTED** [review]

**Comment by nicolasbock** [comment]
Have another look please

**Review by sombrafam — APPROVED** [review]

File: .github/workflows/notofications.yaml (hunk 1 above)
Comment (nicolasbock) [reply .github/workflows/notofications.yaml]: Done

**Pull request merged**